wa_examples/text_wa_canvas_image.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "/wa/translation-alignment/QWxpZ25tZW50Q2h1bmtOb2RlOjE=",
+  "type": "Annotation",
+  "target": [
+    "urn:cts:greekLit:tlg0012.tlg001.msA:1.1-1.7",
```

Comment:
@markpatton one other idea I had, what if we targeted the canvas _and_ image (and use the appropriate selectors).

(otherwise I'll just target the URN and canvas and convert the `FragmentSelector` value to pixels)